test/CMakeLists.txt

```diff
@@ -23,48 +23,63 @@ SET(SYSTEM_SRC
     System/Vector2.cpp
     System/Vector3.cpp
 )
-sfml_add_test(test-sfml-system "${SYSTEM_SRC}" SFML::System)
+sfml_add_test(test-sfml-system "${SYSTEM_SRC}" SFML::System sfml-test-runner-system)
```

Comment:
Can't we remove `SFML::System` since it's already an public dependency of `sfml-test-runner-system`?

Reply:
`SFML::System` is the `DEPENDS` argument or `sfml_add_test` and `sfml-test-runner-system` is the `TEST_RUNNER` argument. This is not a list or dependencies.